core/src/main/scala/org/http4s/StaticFile.scala

```diff
@@ -104,7 +105,7 @@ object StaticFile {
             )
         } else
           blocker
-            .delay(urlConn.getInputStream.close)
+            .delay(Try(urlConn.getInputStream.close()))
```

Comment:
If we used an old-fashioned try-catch block here, we'd save an allocation for a discarded `Try` value, and be able to log a warning on any exceptions.  Without seeing what's being thrown, it's hard to tell whether they'd be noise or something worth addressing.

In any case, returning the `NotModified` here is correct and a worthwhile improvement.

Reply:
From my prospective something like the following snippet would look nicer here:
```
blocker
  .delay(urlConn.getInputStream.close())
  .handleError(_ => ())
  .as( ... )
```
Because it shows intent in a clearer way.

I'm mildly concerned about using `Try` just for swallowing exceptions – it is definitely not what it was designed for (even if it does the job).

Reply:
Yes this looks indeed much nicer.